components/NewCommentForm.js

```diff
@@ -0,0 +1,136 @@
+import styled from "styled-components";
+
+const NewCommentForm = ({ onSubmit, onClose }) => {
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    const comment = event.target.elements.comment.value;
+
+    onSubmit(comment);
+  };
+
+  return (
+    <>
+      <Header>
+        <Title>Add Coment</Title>
+        <CloseButton onClick={onClose}>✕</CloseButton>
+      </Header>
+      <form onSubmit={handleSubmit}>
+        <Content>
+          <TextArea placeholder="Write a comment..." name="comment" />
+        </Content>
+        <Footer>
+          <SubmitButton type="submit">Submit</SubmitButton>
+        </Footer>
+      </form>
+    </>
+  );
+};
+export default NewCommentForm;
+
+const Overlay = styled.aside`
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: rgba(0, 0, 0, 0.7);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+const Popup = styled.section`
+  background: #fff;
+  padding: 20px;
+  border-radius: 12px;
+  width: 400px;
+  max-width: 90%;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+`;
+const Header = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 1px solid #eaeaea;
+  padding-bottom: 10px;
+  margin-bottom: 20px;
+`;
+const Title = styled.h2`
+  margin: 0;
+  font-size: 20px;
+`;
+const CloseButton = styled.button`
+  background: none;
+  border: none;
+  font-size: 24px;
+  cursor: pointer;
+  color: #888;
+  &:hover {
+    color: #000;
+  }
+`;
+const Content = styled.section`
```

Comment:
Make this a label to have basic accessibility:

```suggestion
const Content = styled.label`
```